pkg/model/interface.go

```diff
@@ -129,17 +121,18 @@ func (v *VLLMParam) DeepCopy() VLLMParam {
 		return VLLMParam{}
 	}
 	out := *v
-	out.DistributionParams = make(map[string]string, len(v.DistributionParams))
-	for k, v := range v.DistributionParams {
-		out.DistributionParams[k] = v
-	}
-	out.ModelRunParams = make(map[string]string, len(v.ModelRunParams))
-	for k, v := range v.ModelRunParams {
-		out.ModelRunParams[k] = v
-	}
+	out.DistributionParams = copyStringMap(v.DistributionParams)
```

Comment:
```suggestion
	out.DistributionParams = maps.Clone(v.DistributionParams)
```

This is what you need.